unit_tests/geneteka/generalized_data/ref/births_wiech.json

```diff
@@ -0,0 +1,33 @@
+{
+  "sourceOfData": "geneteka",
+  "sourceType": "record",
+  "recordType": "BirthRegistration",
+  "eventDate": {
+    "yearString": "1876"
+  },
+  "name": {
+    "lastName": "Więch",
+    "forenames": "Paweł Władysław"
+  },
+  "parents": {
+    "father": {
+      "name": {
+        "firstName": "Ludwik",
+        "lastName": "Więch"
+      }
+    },
+    "mother": {
+      "name": {
+        "firstName": "Apolonia",
+        "lastName": "Zalewska"
+      }
+    }
+  },
+  "birthDate": {
+    "dateString": "05.01.1876"
```

Comment:
In the generalized data the dates should be a form that is supported. All numeric dates like this are not because they are ambiguous. So generalizeData could convert this to "5 Jan 1876" for example.

Reply:
Changed